src/TemplateLibrary/TemplateCard.js

```diff
@@ -7,11 +7,21 @@ import TemplateActions from './TemplateActions';
 
 const CardContainer = styled(Card)`
   margin: 10px 0 !important;
+  background-color: ${props => props.color || null} !important;
```

Comment:
same comment from other PR. let's use the default value of `transparent` rather than null. https://www.w3schools.com/cssref/pr_background-color.asp